src/nimgen/external.nim

```diff
@@ -123,18 +123,15 @@ proc runPreprocess*(file, ppflags, flags: string, inline: bool): string =
     if line.strip() != "":
       if line[0] == '#' and not line.contains("#pragma"):
         start = false
-        if sfile in line.replace("\\", "/").replace("//", "/"):
+        if sfile in line.multiReplace([("\\", "/"), ("//", "/")]):
           start = true
```

Comment:
@jyapayne added `sanitizePath()` in a PR which I just merged, we could just reuse that. Will it be better as a template?

https://github.com/genotrance/nimgen/blob/6b38a558034367725dd0564b9ac7d2cfd29b6096/src/nimgen/external.nim#L5-L7